zsl/_state.py

```diff
@@ -0,0 +1,14 @@
+_current_app = None
```

Comment:
The reason for `_current_app` existence is the injector, all code is declared with `@inject` and not `@app.inject`. It not the best solution but works for now and I'll be glad to hear some other idead how to dynamically get current app injector configuration without it.